env.faucet.config.json

```diff
@@ -78,6 +78,32 @@
         "default": "",
         "masked": true,
         "type": "string"
+      },
+      "CAPTCHA_PROVIDER": {
+        "description": "The captcha provider to use. Currently, Prosopo's 'procaptcha' and Google's 'recaptcha' are supported",
+        "default": "procaptcha",
+        "type": "string"
+      },
+      "PROCAPTCHA_MAX_VERIFIED_TIME": {
+        "description": "The maximum time in ms that a user can be verified for",
+        "default": 60000,
+        "type": "number"
+      },
```

Comment:
I think this should belong in the constants of the code.  
It's not a secret, and I can't believe that we will configure it differently for different environments.

Reply:
This is done btw.